notebooks/check_sed_from_dl3.ipynb

```diff
@@ -0,0 +1,837 @@
+{
```

Comment:
<p>Ok, but maybe add a comment that plot_spectrum() defaults to e2dnde and erg cm-2 s-1 for the sed type and unit.</p><p>What you do is correct but may be confusing if this behaviour is not known (as it was my case before looking at the source code of gammapy)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/magic-cta-pipe/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>OK, I will add the comment, thanks!</p><p> </p>